certbot/certbot/util.py

```diff
@@ -539,17 +539,10 @@ def enforce_domain_sanity(domain):
                 )
             )
 
+# there was ip address check : removed for support ip address
```

Comment:
indention of the comment to be aligned with the next lines.
Shall we also drop the following comments, related to the IP address check?

Reply:
here was where old ip check was, I removed but comment left behind